Assets/Scripts/StateTransitions/TransitionManager.cs

```diff
@@ -52,6 +52,18 @@ public static void processTransition(int selected)
                 }
                 break;
 
+            case (GlobalStateTracker.States.ActionMenuing, null):
+                if (selected == 0) //assumed to mean healing move is selected
+                {
+                    GlobalStateTracker.battleState = GlobalStateTracker.CharacterSelect;
```

Comment:
This should be setting battle state to GlobalStateTracker.States.PostActionEntitySelect

PostActionEntitySelect represents the state that the battle scene is in when a character has selected an action and is about to apply it to another entity. In this case, there is a potential healing/buffing action which we need to track in the global state. We then need to see who to apply it to. That is the state we are in, if that makes sense.